x-pack/plugins/security_solution/public/resolver/view/panels/panel_content_utilities.tsx

```diff
@@ -27,8 +27,7 @@ const BetaHeader = styled(`header`)`
  * The two query parameters we read/write on to control which view the table presents:
  */
 export interface CrumbInfo {
-  readonly crumbId: string;
-  readonly crumbEvent: string;
+  readonly [x: string]: string;
```

Comment:
:grey_question: (defer: do do not this until after merge if you decide to) At this point, you could probably just change it to a `Record<string, string>` for clarity.